schemas/context/profile.schema.json

```diff
@@ -86,6 +91,24 @@
           "description":
             "The unit ID within the organization owning the profile. This ID can be used to reference the organization details maintained in another dataset."
         },
+        "xdm:preferredLanguage": {
+          "title": "Preferred Language",
+          "type": "string",
+          "description":
+            "Preferred Language"
+        },
+        "xdm:thumbnail": {
+          "title": "Thumbnail",
+          "type": "string",
+          "description":
+            "Link to profile's picture"
+        },
+        "xdm:emailFormat": {
```

Comment:
format in what format??  Is this a regex?  What is it??

Reply:
Added format details (enum) in latest commit.